Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -344,38 +345,47 @@ bool VolumeVerifier::IsDebugSigned() const
 
 bool VolumeVerifier::ShouldHaveChannelPartition() const
 {
-  const std::unordered_set<std::string> channel_discs{
-      "RFNE01", "RFNJ01", "RFNK01", "RFNP01", "RFNW01", "RFPE01", "RFPJ01", "RFPK01", "RFPP01",
-      "RFPW01", "RGWE41", "RGWJ41", "RGWP41", "RGWX41", "RMCE01", "RMCJ01", "RMCK01", "RMCP01",
+  static constexpr std::array<std::string_view, 18> channel_discs = {
+      "RFNE01", "RFNJ01", "RFNK01", "RFNP01", "RFNW01",
+      "RFPE01", "RFPJ01", "RFPK01", "RFPP01", "RFPW01",
+      "RGWE41", "RGWJ41", "RGWP41", "RGWX41",
+      "RMCE01", "RMCJ01", "RMCK01", "RMCP01",
   };
+  assert(std::is_sorted(channel_discs.cbegin(), channel_discs.cend()));
 
-  return channel_discs.find(m_volume.GetGameID()) != channel_discs.end();
+  return std::binary_search(channel_discs.cbegin(), channel_discs.cend(),
+                            std::string_view(m_volume.GetGameID()));
 }
 
 bool VolumeVerifier::ShouldHaveInstallPartition() const
 {
-  const std::unordered_set<std::string> dragon_quest_x{"S4MJGD", "S4SJGD", "S6TJGD", "SDQJGD"};
-  return dragon_quest_x.find(m_volume.GetGameID()) != dragon_quest_x.end();
+  static constexpr std::array<std::string_view, 4> dqx = {"S4MJGD", "S4SJGD", "S6TJGD", "SDQJGD"};
```

Comment:
With `ssbb` and `dual_layer_discs`, renaming this from `dragon_quest_x` to `dqx` tips the scale from "use the longer/readable name" to "use the abbreviation".

I tend to lean on the "prefer the readable one" side, but I wonder what everyone else thinks about this particular case.
Are we fine with the shorter names here (since they don't really add too much information for the two games, and one could just go and check our wiki for the game ids), or would we prefer the longer ones (mainly for consistency, to present a more uniform style across methods and locals; particularly in this file)?

Reply:
Changed it back to dragon_quest_x for now.